--- zathura-pdf-mupdf-0.4.6/meson.build.orig	2025-10-28 18:28:23.000000000 +0100
+++ zathura-pdf-mupdf-0.4.6/meson.build	2025-10-28 19:36:57.256525823 +0100
@@ -35,7 +35,7 @@
 if not mupdf.found()
   # normal build of mupdf
   mupdf = cc.find_library('mupdf', has_headers: ['mupdf/fitz/version.h', 'mupdf/fitz.h', 'mupdf/pdf.h'], required: true)
-  mupdfthird = cc.find_library('mupdf-third')
+  mupdfthird = cc.find_library('mupdf-third', required: false)
   version_check = '''
 #include <mupdf/fitz/version.h>
 
